modules/scheduler/gke-cluster/variables.tf

```diff
@@ -358,6 +358,24 @@ variable "additional_networks" {
   }))
 }
 
+variable "cluster_reference_type" {
+  description = "How the google_container_node_pool.system_node_pools refers to the cluster. Possible values are: {SELF_LINK, NAME}"
+  default     = "SELF_LINK"
+  type        = string
+}
+
+variable "cluster_availability_type" {
+  description = "Type of cluster availability. Possible values are: {REGIONAL, MULTI_ZONAL}"
```

Comment:
I believe the terminology being used here is wrong and needs to be corrected. I believe the possible availability types are "REGIONAL" and "ZONAL". For either case, you can have node pools that are "ZONAL" or "MULTI_ZONAL" but that's not what the variable is specifying.

- https://cloud.google.com/kubernetes-engine/docs/how-to/creating-a-zonal-cluster#single-zone_versus_multi-zonal
- https://cloud.google.com/kubernetes-engine/docs/how-to/creating-a-regional-cluster#create-regional-single-zone-nodepool

Reply:
Correct, this variable is to specify the cluster location

Reply:
Updated the terminology